vmnet/vmnet.go

```diff
@@ -5,9 +5,10 @@ package vmnet
 import (
 	"fmt"
 	"net"
-	"os"
 	"os/exec"
 	"strings"
+
+	"github.com/docker/machine/libmachine/mcnutils"
```

Comment:
My mistake, I had to make this public to share it and I accidentally put it in the wrong place...